package.json

```diff
@@ -5,18 +5,14 @@
   "homepage": "https://github.com/smlxl/evm.codes",
   "license": "MIT",
   "scripts": {
-    "build": "yarn next build",
+    "build": "next build",
     "build:solcjs-worker": "npx -y browserify -t babelify ./babel/solcWorker2.js > ./public/solcWorker.js",
     "dev": "next dev",
     "format": "prettier --write  --cache .",
     "lint": "eslint --ext js,jsx,ts,tsx .",
     "lint:package": "sort-package-json",
     "typecheck": "tsc --pretty"
   },
-  "resolutions": {
-    "@types/react": "17.0.2",
-    "@types/react-dom": "17.0.2"
-  },
```

Comment:
`resolutions` is a yarn v1 specific feature (that pnpm can read and understand) but I moved it to `pnpm.overrides` to be more explicit.